qa/tasks/cephfs/test_client_recovery.py

```diff
@@ -808,24 +809,35 @@ def test_client_eviction_if_config_is_set(self):
             # it takes time to have laggy clients entries in cluster log,
             # wait for 6 minutes to see if it is visible, finally restart
             # the client
-            tries = 6
-            while True:
-                try:
-                    with self.assert_cluster_log("1 client(s) laggy due to laggy OSDs",
-                                                 timeout=55):
-                        # make sure clients weren't evicted
-                        self.assert_session_count(2)
-                        break
-                except AssertionError:
-                    tries -= 1
-                    if tries:
-                        continue
-                    raise
+            with contextutil.safe_while(sleep=5, tries=6) as proceed:
+                while proceed():
+                    try:
+                        with self.assert_cluster_log("1 client(s) laggy due to"
+                                                     " laggy OSDs",
+                                                     timeout=55):
+                            # make sure clients weren't evicted
+                            self.assert_session_count(2)
+                            break
+                    except (AssertionError, CommandFailedError) as e:
+                        log.debug(f'{e}, retrying')
+
+            # clear lagginess and expect to get the warning cleared
+            self.clear_laggy_params(osd)
+            with contextutil.safe_while(sleep=1, tries=20) as proceed:
```

Comment:
Is the `safe_while` really required? How soon does the OSD gets marked as un-laggy after the call to clear_laggy_params()?

Reply:
> Is the safe_while really required?

not now since sleep is reduced to 1, a simple loop would work too.

> How soon does the OSD gets marked as un-laggy after the call to clear_laggy_params()?

it would be quick since the method to mark it unlaggy involves restarting it(osd); it would then quicky mark the osd in.